README.md

```diff
@@ -2,107 +2,18 @@
 
 here lies the code that runs Odin's right hand bot... 
 
-## Setup
+This bot is used in a server for [theodinproject.com](https://www.theodinproject.com), an open source web development bootcamp, and is used by tens of thousands of Odinites. This bot has been built with NodeJS and [DiscordJS](https://discord.js.org/#/docs/main/stable/general/welcome).
 
-First, fork this repository, then `git clone` the repository. Then, `cd` into the folder and run `npm install` to install all dependencies. Once the repository has been cloned and the dependencies have been installed, you will need to create a `.env` file using `touch .env` or your editor's filesystem, and update it with your own Discord API key. A sample of this file can be found at `.env.sample`, including formatting for your key that will be used when you run the bot. You may skip to the [**Commands**](https://github.com/TheOdinProject/odin-bot-v2/blob/master/README.md#commands) section if you do not want to run your own instance of the Odin-Bot.
+This repo contains the main app for the Odin Project which can be found online at [theodinproject.com](https://www.theodinproject.com). It pulls in lesson and project content from our [curriculum repo](https://github.com/TheOdinProject/curriculum).
```

Comment:
I think "This repo" might be confusing and some may mistake it to mean the bot repo contains the main app code base. I'd be tempted to omit this sentence altogether since it isn't about the bot 🤔

whats your thoughts?

Reply:
Whoops, copied and pasted and forgot to update. Thanks for catching.